src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -33,15 +34,28 @@ export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
         placeholder: 'Enter your reason',
         value: reason,
         onChange: setReason
-      })
+      }),
+      div([
+        h(LabeledCheckbox, {
+          checked: sendCopy,
+          onChange: () => setSendCopy(!sendCopy)
```

Comment:
nit: onChange for the checkbox will pass in the next value
```suggestion
          onChange: setSendCopy
```